yarn.lock

```diff
@@ -8212,6 +8212,39 @@ __metadata:
   languageName: node
   linkType: hard
 
+"csv-generate@npm:^4.2.2":
+  version: 4.2.2
+  resolution: "csv-generate@npm:4.2.2"
+  checksum: f262698c92376261c928779335850c583f5d6ea9f2e476ba476ece7c9fb70bdafe634a4f3024721d44e46b7c50d2135d5932fa383f4fbe48dcaa642635052b2f
+  languageName: node
```

Comment:
Is this the `yarn.lock` file for the `core` package?  Should new packages instead be added just to the `design-system-dashboard-cli` package.json? 

`core` is essentially independent from `design-system-dashboard-cli`. Just theorizing why the tests are failing.

Reply:
Hmm, good point, didn't realize which yarn it had installed into.

Reply:
Ah, it added it to the yarn.lock at the root only because of the way the workspaces feature is setup. The actual dependency is still in the dashboard-cli folder.